prime-dotnet-webapi/Services/EmailRenderingService.cs

```diff
@@ -15,14 +15,10 @@ public class EmailRenderingService : IEmailRenderingService
         private const string MohEmail = "HLTH.HnetConnection@gov.bc.ca";
         private const string ProviderEnrolmentTeamEmail = "Lori.Haggstrom@gov.bc.ca";
```

Comment:
While this isn't part of the PR changes, shouldn't this value be in a ConfigMap, that is, environment variable?  What happens when Lori Haggstrom retires or leaves?

Reply:
Could be a good idea, but having those emails be configurable has never been a requirement

Reply:
Worthwhile for Product Owner to consider?

Reply:
@neophyte57 Can you create a ticket for this comment and then resolve this conversation.